### ISSUE TITLE
Components: enable "copy to clipboard" functionality with keyboard

### ISSUE THREAD

**Issue by ciampo** [WordPress/gutenberg#34795]
Part of #34284

## What

Add keyboard support for the "copy to clipboard" functionality

## Why

Not all users interact with the color picker with a pointer device

## A/C

- [ ] The DOM element that can be currently clicked to copy to clipboard should be flagged as a button, and have a meaningful label (e.g. click to copy to clipboard)
  - [ ] This should work keeping in mind the already present tooltip
- [ ] When the above mentioned DOM element has focus, pressing spacebar should copy the value to the clipboard

**Comment by stokesman** [comment]
Seems like this has been resolved by #41222. Maybe the only qualm would be that the new button’s label ”Copy” is too terse.

**Comment by ciampo** [comment]
Agreed — maybe we could change the text to "Copy to clipboard"?
